specials/SPC18_Matias_SpiesVsMercs.Altis/initPlayerLocal.sqf

```diff
@@ -12,3 +14,16 @@
 ["mission_extract", {
     playSound "SN_Flare_Fired_4";
 }] call CBA_fnc_addEventHandler;
+
+
+//This will hide the Spies markers from the mercs
+
+if (side player != west) then 
+{
+    {deleteMarkerLocal "safehouse"} forEach allMapMarkers;
+    {deleteMarkerLocal "infMark1"} forEach allMapMarkers;
+    {deleteMarkerLocal "infMark2"} forEach allMapMarkers;
+    {deleteMarkerLocal "infMark3"} forEach allMapMarkers;
+    {deleteMarkerLocal "infMark4"} forEach allMapMarkers;
+    {deleteMarkerLocal "infMark5"} forEach allMapMarkers
+};
```

Comment:
You are looping over all the markers, just to delete a specific marker each time.

```sqf
deleteMarkerLocal "safehouse";
for "_i" from 1 to 5 do {
  deleteMarkerLocal format ["infMark%1", _i];
};
```